app/models/apple/episode.rb

```diff
@@ -290,7 +290,7 @@ def episode_create_parameters
             guid: guid,
             title: feeder_episode.title,
             originalReleaseDate: feeder_episode.published_at.utc.iso8601,
-            description: feeder_episode.description_with_default,
+            description: feeder_episode.description_safe,
```

Comment:
naming is hard? safe in this case means "safe to use for Apple integration and the public feed", but maybe just "_safe" is ok?